docs/config/templates/app/indexPage.template.html

```diff
@@ -142,17 +142,19 @@ <h4 class="search-results-group-heading">{{ key }}</h4>
           </div>
         </div>
       </nav>
-      <nav id="navbar-notice" class="navbar navbar-fixed-top">
-        <div class="navbar-inner">
-          <div class="container">
-            <p class="site-notice">
-              AngularJS is now in Long Term Support (LTS) mode: <a href="https://docs.angularjs.org/misc/version-support-status">Find out more</a>. Explore End Of Life (EOL) options <a href="https://blog.angular.io/finding-a-path-forward-with-angularjs-7e186fdd4429">here</a>.
-            </p>
-          </div>
-        </div>
-      </nav>
       <nav id="navbar-sub" class="sup-header navbar navbar-fixed-top" scroll-y-offset-element ng-cloak>
         <div class="container main-grid main-header-grid">
+        <p class="site-notice">
+              AngularJS support has officially ended as of January 2022.
+              <a href="https://docs.angularjs.org/misc/version-support-status">
+                See what ending support means
+              </a> and
+              <a href="https://goo.gle/angularjs-end-of-life">
+                read the end of life announcement
+              </a>.<br>
```

Comment:
```suggestion
              <a href="https://goo.gle/angularjs-end-of-life">read the end of life announcement</a>.<br>
```
This fixes the space before the period:

![Screen Shot 2022-03-10 at 02 16 32](https://user-images.githubusercontent.com/3506071/157609169-ccfb0bd0-a753-4920-b521-1fcd0c0bdb2c.png)

After:
![Screen Shot 2022-03-10 at 02 20 02](https://user-images.githubusercontent.com/3506071/157609340-d3939f02-c653-4112-8a93-fa8f249fd5b3.png)